libs/docs/core/list-byline/examples/list-byline-selection-example/list-byline-selection-example.component.html

```diff
@@ -11,26 +11,44 @@ <h4>With Checkboxes</h4>
         </li>
     }
 </ul>
-<h4>With Radio Buttons</h4>
+<h4 id="with-radio-buttons">With Radio Buttons</h4>
 <ul fd-list [byline]="true" [selection]="true">
     <li fd-list-item [selected]="selectionValue === 'radio1'">
-        <fd-radio-button name="radio1" id="radio1" value="radio1" [(ngModel)]="selectionValue"></fd-radio-button>
+        <fd-radio-button
+            name="radio1"
+            id="radio1"
+            value="radio1"
+            [(ngModel)]="selectionValue"
+            aria-describedby="with-radio-buttons"
+        ></fd-radio-button>
         <span fd-list-thumbnail><fd-icon glyph="calendar"></fd-icon></span>
         <div fd-list-content>
             <div fd-list-title>Title 1</div>
             <div fd-list-byline>Byline 1</div>
         </div>
     </li>
     <li fd-list-item [selected]="selectionValue === 'radio2'">
-        <fd-radio-button name="radio1" id="radio2" value="radio2" [(ngModel)]="selectionValue"></fd-radio-button>
+        <fd-radio-button
+            name="radio1"
+            id="radio2"
+            value="radio2"
+            [(ngModel)]="selectionValue"
+            aria-describedby="with-radio-buttons"
+        ></fd-radio-button>
         <span fd-list-thumbnail><fd-icon glyph="activate"></fd-icon></span>
         <div fd-list-content>
             <div fd-list-title>Title 2</div>
             <div fd-list-byline>Byline 2</div>
         </div>
     </li>
     <li fd-list-item [selected]="selectionValue === 'radio3'">
-        <fd-radio-button name="radio1" id="radio3" value="radio3" [(ngModel)]="selectionValue"></fd-radio-button>
+        <fd-radio-button
+            name="radio1"
+            id="radio3"
+            value="radio3"
+            [(ngModel)]="selectionValue"
+            aria-describedby="with-radio-buttons"
```

Comment:
I think it's more appropriate in this case to have `aria-labelledby` then `aria-describedby`
<img width="809" alt="Screenshot 2024-10-03 at 12 30 19 PM" src="https://github.com/user-attachments/assets/f3f6fc73-031c-4d4f-8525-4569741d2cd2">